source/boundary_traction/interface.cc

```diff
@@ -59,8 +59,375 @@ namespace aspect
+                           "integer or the symbolic name the geometry model in use may have "
+                           "provided for this part of the boundary) "
+                           "and each value must be one of the currently implemented boundary "
+                           "velocity models. ``selector'' is an optional string given as a subset "
+                           "of the letters `xyz' that allows you to apply the boundary conditions "
+                           "only to the components listed. As an example, '1 y: function' applies "
+                           "the type `function' to the y component on boundary 1. Without a selector "
+                           "it will affect all components of the velocity."
+                           "\n\n"
+                           "Note that the no-slip boundary condition is "
+                           "a special case of the current one where the prescribed velocity "
+                           "happens to be zero. It can thus be implemented by indicating that "
+                           "a particular boundary is part of the ones selected "
+                           "using the current parameter and using ``zero velocity'' as "
+                           "the boundary values. Alternatively, you can simply list the "
+                           "part of the boundary on which the velocity is to be zero with "
+                           "the parameter ``Zero velocity boundary indicator'' in the "
+                           "current parameter section."
+                           "\n\n"
+                           "Note that when ``Use years in output instead of seconds'' is set "
+                           "to true, velocity should be given in m/yr. "
+                           "The following boundary velocity models are available:\n\n"
+                           +
+                           std::get<dim>(registered_plugins).get_description_string());
+        prm.declare_entry ("Zero traction boundary indicators", "",
+                           Patterns::List (Patterns::Anything()),
+                           "A comma separated list of names denoting those boundaries "
+                           "on which the velocity is zero."
+                           "\n\n"
+                           "The names of the boundaries listed here can either by "
+                           "numbers (in which case they correspond to the numerical "
+                           "boundary indicators assigned by the geometry object), or they "
+                           "can correspond to any of the symbolic names the geometry object "
+                           "may have provided for each part of the boundary. You may want "
+                           "to compare this with the documentation of the geometry model you "
+                           "use in your model.");
+        // prm.declare_entry ("Tangential velocity boundary indicators", "",
```

Comment:
One of our conventions is to never leave commented code in the repository. In the future it will not be possible to determine why this code is commented. Is it work in progress? Is it outdated? Is it wrong?
We dont need this code, so just remove the whole commented section. Here, and in all other places of your changes.